modules/single-node-asg/main.tf

```diff
@@ -22,6 +22,7 @@ module "service-data" {
   encrypted   = "${var.data_volume_encrypted}"
   kms_key_id  = "${var.data_volume_kms_key_id}"
   snapshot_id = "${var.data_volume_snapshot_id}"
+  extra_tags  = {"createSnapshot" = "${var.name_prefix}"}
```

Comment:
For `createSnapshot`, I would recommend the following:
* we add a new `local.data_volume_name_prefix`, set to `${var.name_prefix}-${var.name_suffix}-data` and to be used here:
```
module "service-data" {
  source      = "../persistent-ebs"
  name_prefix = "${var.name_prefix}-${var.name_suffix}-data"
```
* we add a `local.create_snapshot_tag`, which uses `local.data_volume_name_prefix`
* we use that here in `extra_tags` for the SNASG and DLM policy